javacord-api/src/main/java/org/javacord/api/listener/server/thread/ServerThreadChannelArchiveTimestampChangeListener.java

```diff
@@ -0,0 +1,21 @@
+package org.javacord.api.listener.server.thread;
+
+import org.javacord.api.event.channel.server.thread.ServerThreadChannelArchiveTimestampChangeEvent;
+import org.javacord.api.listener.GloballyAttachableListener;
+import org.javacord.api.listener.ObjectAttachableListener;
+import org.javacord.api.listener.channel.server.ServerChannelAttachableListener;
+import org.javacord.api.listener.server.ServerAttachableListener;
+
+/**
+ * This listener listens to server thread channel archive timestamp change events.
+ */
+public interface ServerThreadChannelArchiveTimestampChangeListener extends ServerAttachableListener,
+        ServerChannelAttachableListener, GloballyAttachableListener, ObjectAttachableListener {
```

Comment:
Please change this in every listener interface. ChannelCategories and ServerVoiceChannel can not have threads so this listener shouldn't be attachable to them.
```suggestion
        ServerTextChannelAttachableListener, GloballyAttachableListener, ObjectAttachableListener {
```